src/mad_spark_alt/core/terminal_renderer.py

```
@@ -0,0 +1,213 @@
+            emoji: Optional emoji prefix
+            status: Status indicator (✓, ✗, etc.)
+            duration: Optional duration in seconds
+        """
+        phase_text = f"{emoji} {phase}" if emoji else phase
+        duration_text = f" ({duration:.1f}s)" if duration is not None else ""
+        
+        if self._fallback_mode:
+            self.console.print(f"{phase_text}... {status}{duration_text}")
+        else:
+            try:
+                status_style = "bold green" if status == "✓" else "bold red"
```

Ruff (0.12.2)

Local variable `status_style` is assigned to but never used

Remove assignment to unused variable `status_style`

(F841)